releasenotes/notes/0.10.2/prepare-0.10.2-d75b2d86be66a6d9.yaml

```diff
@@ -0,0 +1,8 @@
+---
+prelude: >
+  This release is a bugfix release that fixes some issues found in the 0.10.1
```

Comment:
btw, these bugs (except the empty input graph case) were present even in 0.9.0 release

Reply:
Good call, between this and #437 it's definitely more than just regressions in 0.10.x, I tweaked the wording here in https://github.com/Qiskit/retworkx/pull/434/commits/195fc43af1dc2932e2b0fa2f4fd89c7b5db12201